code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -133,12 +133,13 @@
 	name = "brass box"
 	desc = "A huge brass box with several indentations in its surface."

```

Comment:
it's an INdentation though.
There's a DENT - IN the object.

Indent:
"form deep recesses or notches in (a line or surface)." IN A SURFACE